<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1094</width>
    <height>627</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>14</pointsize>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Text Editor</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton {
border-radius: 25px;
padding: 0 8px;
background:#333333;
font: 700 12pt &quot;Consolas&quot;;
height: 50px;
color: white;
}
QPushButton:hover {
border:2px solid;	
border-color: rgb(86, 156, 214);
}

QPushButton:pressed {
         
	background-color: rgb(55, 55, 61);
}
QMainWindow{
background:#252526
}
QLabel{
color:#6a9955;
}
QTextEdit{
background:#1e1e1e;
color:#d4d4d4;
}
QMessageBox{
font: 700 15pt &quot;Courier&quot;;
background:#252526;
color:#6a9955;
width:1200px;
}

</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="openFileButton">
        <property name="text">
         <string>Open File</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="prettifyButton">
        <property name="text">
         <string>Prettify</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="fixButton">
        <property name="text">
         <string>Fix Errors</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="compressButton">
        <property name="text">
         <string>Compress</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="decompressButton">
        <property name="text">
         <string>Decompress</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="jsonButton">
        <property name="text">
         <string>â†’ JSON</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="saveButton">
        <property name="text">
         <string>Save Output</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="decButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font: 700 8pt &quot;Courier New&quot;;
}</string>
        </property>
        <property name="text">
         <string>Aa</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="incButton">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	font: 700 16pt &quot;Courier New&quot;;
}</string>
        </property>
        <property name="text">
         <string>Aa</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QLabel" name="labelFileName">
      <property name="font">
       <font>
        <pointsize>8</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>No File Opened</string>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit_2">
        <property name="font">
         <font>
          <pointsize>12</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="lineWrapMode">
         <enum>QTextEdit::NoWrap</enum>
        </property>
        <property name="tabStopDistance">
         <double>80.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1094</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
